.github/dependabot.yaml

```
@@ -16,8 +15,7 @@ updates:
       - dependency-name: '*'
         update-types: ['version-update:semver-major']
 
-  - package-ecosystem: github-actions
-    directory: blueledger-webapp
+  - package-ecosystem: github-actions    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)